app/views/pageflow/entries/_meta_tags.html.erb

```diff
@@ -0,0 +1,4 @@
+  <meta name="dc.language" content="<%= I18n.locale %>" />
+  <% if entry.keywords.present? %><meta name="keywords" content="<%= entry.keywords %>"><% end %>
+  <% if entry.author.present? %><meta name="author" content="<%= entry.author %>"><% end %>
+  <% if entry.publisher.present? %><meta name="publisher" content="<%= entry.publisher %>"><% end %>
```

Comment:
Before, one could override the partial and set new defaults. Now everything is very configurable, but there is no longer a way to define default values. If the Theming integration is out of scope for you now, we could alternatively allow setting defaults in `Pageflow.conig` and fall back to these here. For the editor, we can pass values from the config in the [config seed data](https://github.com/codevise/pageflow/blob/master/app/views/pageflow/config/_editor_seeds.json.jbuilder) which are accessible in the editor [via `pageflow.config`](https://github.com/codevise/pageflow/blob/master/app/assets/javascripts/pageflow/editor/initializers/setup_config.js). Then we can display placeholders in the text boxes.